repository-data/application/src/main/resources/hcm-content/content/documents/corporate-website/publication-system/ci-hub.yaml

```diff
@@ -43,14 +43,16 @@
       hippostd:state: draft
       hippostdpubwf:createdBy: admin
       hippostdpubwf:creationDate: 2018-01-23T14:54:36.279Z
-      hippostdpubwf:lastModificationDate: 2018-01-24T23:19:37.443Z
+      hippostdpubwf:lastModificationDate: 2018-01-26T14:13:32.852Z
```

Comment:
can we make sure that this file is "empty" in the "application" submodule. We can keep example content in "development" submodule.